lib/vanagon/component.rb

```diff
@@ -215,7 +217,7 @@ def get_source(workdir) # rubocop:disable Metrics/AbcSize
         @dirname = './'
 
         # If there is no source, there is nothing to do to extract
-        @extract_with = ': no source, so nothing to extract'
+        extract_with << ': no source, so nothing to extract'
```

Comment:
I think this could go away, but I think the makefile is still going to be a mess because of https://github.com/puppetlabs/vanagon/blob/ae1f319dbd48ed20f4e65929cf71b2d2208f67da/lib/vanagon/component/source/local.rb#L82